### ISSUE TITLE
Android LOD - Library - The tooltip 'Remove pin from my library page' remains visible

### ISSUE THREAD

**Issue by pcenov** [learningequality/kolibri#11441]
## Observed behavior
After pinning/unpinning a library the tootltip remains visible on the page unless I tap somewhere. If I scroll up the tooltip overlaps the top navigation bar and looks weird.

## Expected behavior
The pin should either not be displayed at all or disappear.

## Steps to reproduce the issue
1. Install the latest APK build from https://github.com/learningequality/kolibri/pull/11419 and setup a learn-only device on an Android device.
2. Go to the Library page and pin/unpin a library.

## More information

![2023-10-20_17-39-45](https://github.com/learningequality/kolibri/assets/79847249/3f54840c-7f1f-48e5-86d0-02af977c6cdc)

## Usage Details
Kolibri 0.16beta6
Android app

**Comment by pcenov** [comment]
@radinamatic

**Comment by marcellamaki** [comment]
The first task might be to see if this behavior is replicable in other places where a tooltip is used. If it appears only in this situation, and not in other places where the component is used, we can fix within the context of this bug. If it appears in other places, however, the issue may require making changes in the `<KTooltip>` component, which is part of the [Kolibri Design System](https://github.com/learningequality/kolibri-design-system), rather than actually making changes in the Kolibri Library page.

**Comment by nick2432** [comment]
can i work on this?

**Comment by bjester** [comment]
Hi @nick2432, I just assigned you to a different issue. I see you're now assigned 3 issues, so when those others are completed and fixes merged, please comment again on this issue and we can assign you. Thank you for your interest in contributing

**Comment by nick2432** [comment]
@bjester  , yes, I noticed I have two assigned issues. Thank you for your attention, and I apologize for any inconvenience caused.

**Comment by AlexanderBlanchardAC** [comment]
Hi, 
If this issue is still unresolved, may I be assigned it?
Cheers,
Alex

**Comment by MisRob** [comment]
@AlexanderBlanchardAC Hi Alex, yes, thank you for volunteering

**Comment by rtibbles** [comment]
Unassigning as it has been over 6 months with no activity.

**Comment by josephinoo** [comment]
Can I get this issue assigned? 🙂

**Comment by AlexVelezLl** [comment]
Thanks for your interest in contributing to this issue @josephinoo! I will assign this to you :)

**Comment by yashhash2** [comment]
Hi @AlexVelezLl If @josephinoo isn't working on this issue I would love to be assigned ; )

**Comment by josephinoo** [comment]
Hi @yashhash2, yes I am working on this issue, I will upload my changes soon. Happy new year 🚀

**Comment by Sanchit-100** [comment]
Hi @AlexVelezLl.
Hi @josephinoo . I would like to know if you are working on this issue.
If you are busy otherwise, I would like to get this assigned.
Thanks

**Comment by MisRob** [comment]
Hi @Sanchit-100, we usually give people a month (and more if requested) which was the case here already and since we didn't hear back from @josephinoo, I will re-assign to you. Thanks for checking-in first!

**Comment by Sanchit-100** [comment]
Hey @pcenov @MisRob 
As mentioned by @marcellamaki in the [comment](https://github.com/learningequality/kolibri/issues/11441#issuecomment-1783094202), the issue is actually replicable in other places as well, where `KTooltip` component is used.

I checked the same for the TotalPoints component in AppBar. Here also, the tooltip doesn't disappear until I click somewhere else. Thus changes might be required in [Kolibri Design System](https://github.com/learningequality/kolibri-design-system) repo to fix this issue everywhere, or could be temporarily fixed for just these context.
I would love to fix the same in the KDS repo. If so, shall I raise an issue for the same there?

<img src="https://github.com/user-attachments/assets/5eab3d6e-b00a-47c6-aafb-b2f9fba408ed" alt="Description of Image" width="300" height="700">

**Comment by MisRob** [comment]
Hi @Sanchit-100, yes, it'd be best to fix this in KDS (Kolibri Design System) and you are welcome to open a pull request. Thank you.

**Comment by MisRob** [comment]
Unassigning @Sanchit-100 due to no activity for 1 month. @Sanchit-100, if you're still interested in this issue or already have work in progress, please message us here, and we'll assign you again. Thank you!
